server/odc-service/src/main/java/com/oceanbase/odc/service/integration/ExternalSqlInterceptor.java

```diff
@@ -109,7 +109,11 @@ public boolean doPreHandle(@NonNull SqlAsyncExecuteReq request, @NonNull SqlAsyn
                         .ifPresent(rule -> {
                             Rule violationRule = new Rule();
                             RuleViolation violation = new RuleViolation();
+                            violation.setText(request.getSql());
```

Comment:
if violates the external SQL interception rule, then put all SQLs into one line, and the highlight is all the SQLs.